### DISCUSSION TITLE
Config Help - Cannot Access Addon

### DISCUSSION THREAD

**Discussion by AshGSmith** [hassio-addons/app-nut#377] [Q&A]
I'm tearing my hair out trying to get this working so I'm really hoping someone can help! 

I've got a Powercool 1500va UPS and I've confirmed it's on the compatible devices list. I've got the add-on installed and running with the configuration below:
`users:
      - username: xxxxx
         password: xxxxx
         instcmds:
               - all
         actions: []
devices:
      - name: PowercoolUPS
        driver: nutdrv_qx
        port: auto
        config: []
mode: netserver
shutdown_host: false`

All looks ok, but when I try and add the integration to link to the add-on it can't find it when using localhost, and when I use the IP address of my Home Assistant instance, I get the following error (IP address removed for security):
Connection error: [Errno 111] Connect call failed ('IPADDRESS', 3493)

I've spent far too much time already trying to get this working and being stubborn not wanting to ask for help, so if someone can come along and save the day I'll be forever grateful! 

Thanks,
Ash

**Comment by guineapirates** [comment]
dont use ip or localhost but "a0d7b954-nut"
